OrangeDesignSystem/Sources/OrangeDesignSystem/Modules/About/ODSAboutModule.swift

```diff
@@ -92,7 +94,12 @@ public struct ODSAboutModule<TermsOfService>: View where TermsOfService: View {
                 .padding(.bottom, ODSSpacing.s)
         }
         .listStyle(.plain)
-        .navigationTitle(°°"modules.about.title")
+        .navigationTitle(title)
+        .onAppear {
```

Comment:
We should aply it everywhere in (demo app, in lib) ?

Reply:
Yes, maybe. Each buggy case will be listed in dedicated card. So we may apply it in both lib and app.